containers/ecr-viewer/src/app/api/tests/listEcrDataService.test.tsx

```diff
@@ -154,166 +90,117 @@ describe("listEcrDataService", () => {
+          {
+            ecr_id: "1234",
+            date_created: new Date("2024-06-21T12:00:00Z"),
+            patient_birth_date: new Date("11/07/1954"),
+            patient_name_first: "Billy",
+            patient_name_last: "Bob",
+            report_date: new Date("06/21/2024 8:00 AM EDT"),
+            reportable_condition: "stuff",
+            rule_summary: "yup",
+            data: "",
+            data_link: "",
+            data_source: "DB",
+          },
+        ]),
+      );
 
-  describe("getS3", () => {
-    it("should return data from S3", async () => {
-      process.env.SOURCE = "s3";
-      s3Mock.on(ListObjectsV2Command).resolves({
-        Contents: [
-          { Key: "id1", LastModified: new Date("2024-06-23T12:00:00Z") },
-        ],
-      });
-      const actual = await listEcrData();
+      let startIndex = 0;
+      let itemsPerPage = 25;
+      const actual: EcrDisplay[] = await listEcrData(startIndex, itemsPerPage);
 
+      expect(database.manyOrNone).toHaveBeenCalledExactlyOnceWith(
+        "SELECT ecr_id, patient_name_first, patient_name_last, patient_birth_date, report_date, reportable_condition, rule_summary, date_created FROM fhir_metadata order by date_created DESC OFFSET 0 ROWS FETCH NEXT 25 ROWS ONLY",
+      );
       expect(actual).toEqual([
         {
-          dateModified: "06/23/2024 8:00 AM EDT",
-          ecrId: "id1",
+          date_created: "06/21/2024 8:00 AM EDT",
+          ecrId: "1234",
+          patient_date_of_birth: "11/07/1954",
+          patient_first_name: "Billy",
+          patient_last_name: "Bob",
+          patient_report_date: "06/21/2024 8:00 AM EDT",
+          reportable_condition: "stuff",
+          rule_summary: "yup",
         },
       ]);
     });
-    it("should fetch FHIR Metadata", async () => {
-      process.env.SOURCE = "s3";
-      process.env.NEXT_PUBLIC_NON_INTEGRATED_VIEWER = "true";
-      s3Mock.on(ListObjectsV2Command).resolves({
-        Contents: [
-          { Key: "id1", LastModified: new Date("2024-06-23T12:00:00Z") },
-        ],
-      });
-      database.manyOrNone<{ ecr_id: string; date_created: string }> = jest.fn(
-        () =>
-          Promise.resolve([
-            {
-              ecr_id: "id1",
-              date_created: "2024-06-21T12:00:00Z",
-              patient_name_last: "lnam",
-              patient_birth_date: "1990-01-01T05:00:00.000Z",
-              report_date: "2024-06-20T04:00:00.000Z",
-              reportable_condition: "sick",
-            },
-          ]),
-      );
 
-      const actual = await listEcrData();
+    it("should get data from the fhir_metadata table", async () => {
+      database.manyOrNone<{
+        ecr_id: string;
+        date_created: string;
+        patient_birth_date: string;
+        patient_name_first: string;
+        patient_name_last: string;
+        report_date: string;
+        reportable_condition: string;
+        rule_summary: string;
+      }> = jest.fn(() =>
```

Comment:
same as above, can this type be replaced by `EcrMetadataModel`?     
```suggestion
      database.manyOrNone<EcrMetadataModel> = jest.fn(() =>
```